spec/integration/post_index_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'rails_helper'
+
+RSpec.describe 'Post index view', type: :system do
+  before :all do
+    @tom = User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico')
+    @lilly = User.create(name: 'Lilly', photo: '#', bio: 'Teacher from Poland')
+    @post1 = Post.create(title: 'post1', text: 'text1', author_id: @tom.id)
+    @post2 = Post.create(title: 'post2', text: 'text2', author_id: @tom.id)
+    @post3 = Post.create(title: 'post3', text: 'text3', author_id: @tom.id)
+    @post4 = Post.create(title: 'post4', text: 'text4', author_id: @tom.id)
+    @post5 = Post.create(title: 'post5', text: 'text5', author_id: @lilly.id)
+    @comment1 = Comment.create(text: 'comment1', author_id: @lilly.id, post_id: @post1.id)
+    @comment2 = Comment.create(text: 'comment2', author_id: @lilly.id, post_id: @post1.id)
+    @comment3 = Comment.create(text: 'comment3', author_id: @lilly.id, post_id: @post1.id)
+    @comment4 = Comment.create(text: 'comment4', author_id: @lilly.id, post_id: @post1.id)
+    @comment5 = Comment.create(text: 'comment5', author_id: @lilly.id, post_id: @post1.id)
+    @comment6 = Comment.create(text: 'comment6', author_id: @lilly.id, post_id: @post1.id)
+  end
+  it 'shows the profile picture of the user' do
+    visit user_posts_path(@tom)
+    expect(page).to have_css("img[src='https://unsplash.com/photos/F_-0BxGuVvo']")
+  end
+
+  it 'shows the user\'s name' do
+    visit user_posts_path(@tom)
+    expect(page).to have_content('Tom')
```

Comment:
- [ ] Please make sure that all your test is passing, Currently almost all your test are failing so kindly fix them :pray:

![image](https://github.com/HanaSabih/BlogApp/assets/92303891/8454830e-c20d-4485-941f-b04781f71d59)